src/platform-includes/crons/troubleshooting/node.mdx

```diff
@@ -0,0 +1,3 @@
+### How Do I Send an Attachment With a Check-in (Such as a Log Output)?
+
+Attachments aren't supported by our Node SDK yet. For now, you can use the [check-in attachments API](/product/crons/getting-started/http/#check-in-attachment-optional).
```

Comment:
This content doesn't appear in the preview. 

I think you missed adding `<PlatformSection supported={["python", "php", "node"]}>` 

to `src/platforms/common/crons/troubleshooting-crons.mdx`

Reply:
Good catch! Fixed with 8a30507c72fe0ef06516d5b67122be3a1ca7ef4f